Makefile

```diff
@@ -4,12 +4,18 @@ all:
 	$(MAKE) build
 
 .PHONY: get-deps
-get-deps:
-	pip install -r requirements.txt
+get-deps: get-deps-pip get-deps-proto get-deps-webui
 	$(MAKE) -C master $@
 	$(MAKE) -C agent $@
```

Comment:
It makes sense to me that you would put these two along with `make -C proto get-deps` under a target like "get-deps-go" or something.  There's no reason, for instance, why `get-debs-webui` should be required to finish before master and agent deps.